smarts/core/smarts.py

```diff
@@ -1423,7 +1423,7 @@ def _process_collisions(self):
                 actor_id = self._vehicle_index.owner_id_from_vehicle_id(collidee.id)
                 # TODO: Should we specify the collidee as the vehicle ID instead of
                 #       the agent/social ID?
-                collision = Collision(collidee_id=actor_id)
+                collision = Collision(collidee_id=actor_id or collidee.id)
```

Comment:
Perhaps choose one instead of using `or`.

Reply:
@Adaickalavan do you prefer knowing the agent id (or other type of owner) or the vehicle id?

Reply:
I have made it always the vehicle name.